modules/pipeline/dbclient/op_pipeline_base.go

```diff
@@ -59,6 +60,7 @@ func (client *Client) ListPipelineBaseWithDefinitionByIDs(pipelineIDs []uint64,
 func (client *Client) CreatePipelineBase(base *spec.PipelineBase, ops ...SessionOption) error {
 	session := client.NewSession(ops...)
 	defer session.Close()
+	base.ID = uuid.SnowFlakeIDUint64()
```

Comment:
put `base.ID = uuid.SnowFlakeIDUint64()` with `session.InsertOne` in the same block.